LifeCycleAssessment_Engine/Compute/EvaluateExplicitBulkByVolume.cs

```diff
@@ -0,0 +1,85 @@
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System.Linq;
+using System.ComponentModel;
+using BH.oM.Base;
+using BH.oM.Reflection.Attributes;
+using BH.oM.LifeCycleAssessment;
+using BH.oM.LifeCycleAssessment.MaterialFragments;
+using BH.oM.LifeCycleAssessment.Results;
+using BH.Engine.Base;
+using BH.oM.Physical.Elements;
+using BH.oM.Dimensional;
+
+namespace BH.Engine.LifeCycleAssessment
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /****   Public Methods                          ****/
+        /***************************************************/
+
+        [Description("This method calculates the total value of the selected environmental impact metric.\n" +
+            "This compute method functions by multiplying the object's SolidVolume by the EPD environmental impact metric.")]
+        [Input("explicitBulk", "ExplicitBulk objects are any object not currently formatted as a standard BHoM object (i.e. bars, floors, or panels), but can host a MaterialComposition for material based analysis along with a Volume quantity attribute.")]
+        [Input("field", "Filter the provided EnvironmentalProductDeclaration by selecting one of the provided metrics for calculation.")]
+        [Output("quantity", "The total quantity of the desired metric based on the EnvironmentalProductDeclarationField")]
+        public static GlobalWarmingPotentialResult EvaluateExplicitBulkByVolume(ExplicitBulk explicitBulk = null, EnvironmentalProductDeclarationField field = EnvironmentalProductDeclarationField.GlobalWarmingPotential)
```

Comment:
Would have assumed that we should not need these methods? Why cant we just treat things as IElementM?

Reply:
I was going for a more explicit UI functionality thinking that some may want to only work with certain types of objects. But agreed that these methods are unnecessary as they open us up to potentially more unnecessary risk and code management. Fortunately, an easy problem to fix! 

Since it's UI messages I'm after, I'll simply provide notes where needed in the next commit. Thanks @IsakNaslundBh